docs/introduction.mdx

```diff
@@ -84,9 +84,10 @@ const sum = atom((get) => atoms.map(get).reduce((acc, count) => acc + count))
 ```
 
 ### Derived async atoms
+
 <img src="https://img.shields.io/badge/-needs_suspense-black" alt="needs suspense" />
```

Comment:
I guess we want the badge in the header. Does this work?
```suggestion
### Derived async atoms <img src="https://img.shields.io/badge/-needs_suspense-black" alt="needs suspense" />
```

Reply:
Hmm, it doesn't work anyway. Let's leave it for now.
<img width="315" alt="image" src="https://user-images.githubusercontent.com/490574/160219473-cc5566db-b5fb-4960-b3d5-20df1a0a4c98.png">